tests/llm/fixtures/test_ask_holmes/55_fetch_error_logs_with_errors/test_case.yaml

```diff
@@ -0,0 +1,7 @@
+user_prompt: "post last 10 logs that contain \"ERROR\" from payment-service in production-55 namespace"
+expected_output: "ERROR: Failed to connect to database"
+before_test: 'kubectl apply -f ./manifest.yaml'
```

Comment:
please run the eval 100 times
I'm a bit concerned that the expected output here it too vague.